### ISSUE TITLE
Game fails to load after saving key mapping in RetroArch

### ISSUE THREAD

**Issue by Mik027** [cizia64/CrossMix-OS#202]
Last time, I wanted to configure the key bindings for a DOS game. I saved the mapping and requested in the RetroArch menu to keep this layout for this specific game. Now, whenever I try to launch this game (it doesn't happen with others), RetroArch starts without loading the game, without a Core, without anything !

I haven't noted it down, but several times strange things have happened to me in the same way.
For example, after browsing through RetroArch's settings and then quitting, I lost the shaders for the GBA.

In short, just problems coming out of nowhere, even without a direct link to the action that was performed (at least in appearance).

I'm starting to get fed up with this instability. I've formatted and recopied the files on the SD card three times now to start from scratch and erase the problems.

Have you ever been bothered by something similar ?

**Comment by Nevrdid** [comment]
There are 3 things you can save and you not save them the same place ( 4 with save state): retroarch config, core config and remaping. Looks you just saved a retroarch config override instead of a remapping. However, having your /tmp/.lastlog should help understand better the issue as i may be wrong.
